tests/erp/test_browser.py

```diff
@@ -452,7 +452,9 @@ def mock_geocode(*args, **kwargs):
             "cheminement_ext_reperage_marches": True,
             "cheminement_ext_main_courante": True,
             "cheminement_ext_rampe": "aucune",
-            "cheminement_ext_pente": "aucune",
+            "cheminement_ext_pente": True,
```

Comment:
Hmm, for forms I think the HTML field value should be `"on"` or `"off"`, not straight booleans. I might work here because a truthy value is expected but in the case of a `False` you may have bad surprises (maybe check it using `False` in the test, then switch it to `"off"` if it fails).

Reply:
Sorry it only applies to checkboxes.